.gitattributes

```diff
@@ -0,0 +1,93 @@
+## GITATTRIBUTES FOR WEB PROJECTS
```

Comment:
Taken from Meadowlark repo

Reply:
Its missing a lot of .net specific stuff, like ignoring `obj/` and `bin/`.

Reply:
I think you mean .gitignore 🙂

Reply:
Yes, good point, I thought I was looking at something different.